src/UI/App.jsx

```diff
@@ -509,8 +519,10 @@ export default class App extends React.Component {
 	showChat = () => {
 		Logger.debug("Show chat, registering device");
 
-		this.setState(() => ({showChat: true}));
-		localStorage.setItem("messengerOpenState", messengerState.open);
+		this.setState(() => ({
+			showChat: true,
+			messengerOpenState: MessengerOpenState.open,
```

Comment:
Door dit zo aan te roepen zet je het dan ook direct in de storage? Volgens mij niet namelijk, waar zet je nu in de storage wat de state is na het indrukken van de launcher?

Reply:
door de `setState()` wordt `shouldComponentUpdate()` gedraaid, die heeft de `localStorage.setItem()`, zie [hier](https://github.com/parley-messaging/web-library/pull/41/files/dd8639ca77911208c5a57f83d9be7553f1d0af97..4d2d099b4387e1fb3a4f132545923e54278dc8e9#diff-0abef088b3f6260f845818ba7eef7e55a43e70e212c423ebaa54066f4c94aba7R311)

Reply:
Aight ja dat had ik over het hoofd gezien maar verklaard de werking prima! Thanks.